metal/datasource_metal_port_test.go

```diff
@@ -0,0 +1,94 @@
+package metal
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccMetalPort_ByName(t *testing.T) {
+
+	rs := acctest.RandString(10)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testPortConfig_ByName(rs),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"data.metal_port.test", "bond_port", "false"),
+				),
+			},
+		},
+	})
+}
+
+func testPortConfig_ByName(name string) string {
+	return fmt.Sprintf(`
+
+resource "metal_project" "test" {
+    name = "tfacc-port-%s"
+}
+
+resource "metal_device" "test" {
+  hostname         = "tfacc-test-device-port"
+  plan             = "c3.medium.x86"
+  facilities       = ["sv15"]
```

Comment:
Unless we are testing `facility` workflows, we should test with metros for a better success rate in tests (more availability).

Reply:
:+1: